great_expectations/core/expectation_diagnostics/expectation_diagnostics.py

```diff
@@ -163,11 +184,34 @@ def _count_positive_and_negative_example_cases(
         negative_cases: int = 0
 
         for test_data_cases in examples:
-            for test in test_data_cases["tests"]:
-                if test["output"]["success"] is True:
-                    positive_cases += 1
-                elif test["output"]["success"] is False:
-                    negative_cases += 1
+            try:
+                for test in test_data_cases["tests"]:
+                    if test["output"]["success"] is True:
+                        positive_cases += 1
+                    elif test["output"]["success"] is False:
+                        negative_cases += 1
+            except:
+                # If there's no "success" key, should it count for negative_cases?
```

Comment:
I'm inclined to say yes -- if a negative case is "this is what this expectation DOES NOT DO" then a user setting up an example intended to error out absolutely satisfies that. 

@abegong @kenwade4 @talagluck thoughts?

Reply:
After looking at it closer, the point of this function is to see how many test examples are explicitly checking for success == True / False (to make sure they have at least 1 negative and 1 positive test case defined. Those should not be counted.

I added DEVREL-1379 to JIRA to look into the old core Expectations that have at least 1 test example that doesn't specify "success" in the output.